docs/developer/other/API.md

```diff
@@ -106,3 +106,47 @@ Example response:
   }
 }
 ```
+###Portlet Preferences
```

Comment:
There needs to be a space between `###` and `Portlet Preferences`

Reply:
done